common/web/types/src/main.ts

```diff
@@ -31,3 +31,6 @@ export { VariableParser, MarkerParser } from './ldml-keyboard/pattern-parser.js'
 export { ElementString } from './kmx/kmx-plus/element-string.js';
 
 export { USVString, CasingForm, CasingFunction, TextWithProbability, LexiconTraversal, LexicalModel, LexicalModelPunctuation, Transform, Suggestion, Reversion, Keep, SuggestionTag, Context, Distribution, Outcome, WithOutcome, ProbabilityMass, Configuration, Capabilities, WordBreakingFunction, Span } from './lexical-model-types.js';
+
+export { CacheTag, ComplexKeyboardStore, EncodedVisualKeyboard, LayoutSpec, KeyboardObject } from './keyboard-object.js';
+export { OutputTarget } from './outputTarget.interface.js';
```

Comment:
I am becoming increasingly uncomfortable with how many global types we are exporting from common/web/types now, with little-to-no structure, e.g. `Configuration` or `Capabilities` are completely unhelpful global type names. I think that the lexical model types should be exported inside a container, `export * as LexicalModelTypes from './lexical-model-types.js'` and the consumers updated accordingly. (Yeah, this should be a separate PR, but in the 18.0 cycle because this change hasn't landed in stable yet.)

Similarly with this keyboard object, it would be good to export it with a container name. And as discussed above, I think `OutputTarget` should be kept inside KMW.

```suggestion
export * as LexicalModelTypes from './lexical-model-types.js';

export * as KmwKeyboard from './keyboard-object.js';
```

There are a few other similar instances of scope creep above, with the KMX+/LDML Keyboard types, which we could consider tidying up as well -- see the TODO on line 28!

Reply:
> (Yeah, this should be a separate PR, but in the 18.0 cycle because this change hasn't landed in stable yet.)

Tracking in #12516

Reply:
`OutputTarget` moved back to KMW.